docsrc/getting_started/jetpack.rst

```diff
@@ -0,0 +1,117 @@
+
+You can only install the dev components that you require: ex, tensorrt-dev would be the meta-package for all TRT development or install everthing.
+
+.. code-block:: sh
+    # install all the nvidia-jetpack dev components
+    sudo apt update
+    sudo apt install nvidia-jetpack
+
+Ensure you have cuda 12.6 installed(this should be installed automatically from nvidia-jetpack)
+
+.. code-block:: sh
+
+    # check the cuda version
+    nvcc --version
+    # if not installed or the version is not 12.6, install via the below cmd:
+    sudo apt update
+    sudo apt install cuda-toolkit-12-6
+
+Ensure libcusparseLt.so exists at /usr/local/cuda/lib64/:
+
+.. code-block:: sh
+
+    # if not exist, download and copy to the directory
+    wget https://developer.download.nvidia.com/compute/cusparselt/redist/libcusparse_lt/linux-sbsa/libcusparse_lt-linux-sbsa-0.5.2.1-archive.tar.xz
+    tar xf libcusparse_lt-linux-sbsa-0.5.2.1-archive.tar.xz
+    sudo cp -a libcusparse_lt-linux-sbsa-0.5.2.1-archive/include/* /usr/local/cuda/include/
+    sudo cp -a libcusparse_lt-linux-sbsa-0.5.2.1-archive/lib/* /usr/local/cuda/lib64/
+
+
+Build torch_tensorrt
+~~~~~~~~~~~~~~
+
+
+Install bazel
+
+.. code-block:: sh
+
+    wget -v https://github.com/bazelbuild/bazelisk/releases/download/v1.20.0/bazelisk-linux-arm64
+    sudo mv bazelisk-linux-arm64 /usr/bin/bazel
+    chmod +x /usr/bin/bazel
+
+Install pip and required python packages
+
+.. code-block:: sh
+
+    # install pip
+    sudp apt install python3-pip
+
+.. code-block:: sh
+
+    # install setuptools with the version less than 71.*.* 
+    python -m pip install setuptools==70.2.0
+
+.. code-block:: sh
+
+    # install torch
+    wget https://developer.download.nvidia.cn/compute/redist/jp/v61/pytorch/torch-2.5.0a0+872d972e41.nv24.08.17622132-cp310-cp310-linux_aarch64.whl
+    python -m pip install torch-2.5.0a0+872d972e41.nv24.08.17622132-cp310-cp310-linux_aarch64.whl
+
+    # install torchvision
+    # currently it has not available yet for JetPack 6.1, it should be available in future
+
+
+Build and Install torch_tensorrt wheel file
+
+
+Since torch_tensorrt version has dependencies on torch version. torch version supported by JetPack6.1 is from DLFW 24.08/24.09(torch 2.5.0).
+
+Please make sure to build torch_tensorrt wheel file from source release/2.5 branch
+(TODO: lanl to update the branch name once release/ngc branch is available)
+
+.. code-block:: sh
+
+    cuda_version=$(nvcc --version | grep Cuda | grep release | cut -d ',' -f 2 | sed -e 's/ release //g')
+    export TORCH_INSTALL_PATH=$(python -c "import torch, os; print(os.path.dirname(torch.__file__))")
+    export SITE_PACKAGE_PATH=${TORCH_INSTALL_PATH::-6}
+    export CUDA_HOME=/usr/local/cuda-${cuda_version}/
+    # replace the MODULE.bazel with the jetpack one
+    cat toolchains/jp_workspaces/MODULE.bazel.tmpl | envsubst > MODULE.bazel
+    # build and install torch_tensorrt wheel file
+    python setup.py --use-cxx11-abi install --user
```

Comment:
can we just use `USE_CXX11_ABI=1 python -m pip install --user`? Then all the deps will get handled by pip

Reply:
I tried with pip install however it got the following error:
seems like try to build Tensorrt, I don't know how to fix it however setup.py is working as expected.
```
ubuntu@ubuntu:~/Desktop/lan/git/TensorRT$ USE_CXX11_ABI=1 python -m pip install --user .
Processing /home/ubuntu/Desktop/lan/git/TensorRT
  Installing build dependencies ... error
  error: subprocess-exited-with-error

  × pip subprocess to install build dependencies did not run successfully.
  │ exit code: 1
  ╰─> [41 lines of output]
      Collecting setuptools>=68.0.0
        Using cached setuptools-75.1.0-py3-none-any.whl.metadata (6.9 kB)
      Collecting packaging>=23.1
        Using cached packaging-24.1-py3-none-any.whl.metadata (3.2 kB)
      Collecting wheel>=0.40.0
        Using cached wheel-0.44.0-py3-none-any.whl.metadata (2.3 kB)
      Collecting ninja>=1.11.0
        Using cached ninja-1.11.1.1-py2.py3-none-manylinux2014_aarch64.manylinux_2_17_aarch64.whl.metadata (5.3 kB)
      Collecting pyyaml>=6.0
        Using cached PyYAML-6.0.2-cp310-cp310-manylinux_2_17_aarch64.manylinux2014_aarch64.whl.metadata (2.1 kB)
      Collecting cffi>=1.15.1
        Using cached cffi-1.17.1-cp310-cp310-manylinux_2_17_aarch64.manylinux2014_aarch64.whl.metadata (1.5 kB)
      Collecting typing-extensions>=4.7.0
        Using cached typing_extensions-4.12.2-py3-none-any.whl.metadata (3.0 kB)
      Collecting future>=0.18.3
        Using cached future-1.0.0-py3-none-any.whl.metadata (4.0 kB)
      Collecting tensorrt-cu12==10.3.0
        Using cached tensorrt-cu12-10.3.0.tar.gz (18 kB)
        Preparing metadata (setup.py): started
        Preparing metadata (setup.py): finished with status 'error'
        error: subprocess-exited-with-error

        × python setup.py egg_info did not run successfully.
        │ exit code: 1
        ╰─> [6 lines of output]
            Traceback (most recent call last):
              File "<string>", line 2, in <module>
              File "<pip-setuptools-caller>", line 34, in <module>
              File "/tmp/pip-install-ewd6tmk6/tensorrt-cu12_db2ff6366355445989f29692870c7b16/setup.py", line 67, in <module>
                raise RuntimeError("TensorRT currently only builds wheels for x86_64 processors")
            RuntimeError: TensorRT currently only builds wheels for x86_64 processors
            [end of output]

        note: This error originates from a subprocess, and is likely not a problem with pip.
      error: metadata-generation-failed

      × Encountered error while generating package metadata.
      ╰─> See above for output.

      note: This is an issue with the package mentioned above, not pip.
      hint: See above for details.
      [end of output]
```

  note: This error originates from a subprocess, and is likely not a problem with pip.
error: subprocess-exited-with-error

Reply:
I see